### PR TITLE
Prepare 2.1.0 release with stylelint 8.0 support

### LINKED ISSUE #12: support stylelint 8.0
not sure what work is necessary here, but our current peerDep is `^7.9.0`.

### PR DESCRIPTION
**Pull request by giladgray**
#### Fixes #12 

- Update all dependencies
- Also reverts `selector-max-universal` limit to 0
- Bump package version to 2.1.0 so we're ready to release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,15 +23,15 @@
...
   "license": "Apache-2.0",
   "dependencies": {
-    "stylelint-config-standard": "^16.0.0",
+    "stylelint-config-standard": "^17.0.0",
     "stylelint-order": "^0.3.0"
   },
   "devDependencies": {
-    "stylelint": "7.9.0",
-    "stylelint-scss": "1.4.3"
+    "stylelint": "^8.0.0",
```
Comment (adidahiya) [inline package.json]: we don't have a lockfile, so you should pin this version down. you can add yarn in a future PR.

File: package.json [hunk 2]
```diff
@@ -23,15 +23,15 @@
...
+    "stylelint-scss": "^1.5.1"
   },
   "peerDependencies": {
-    "stylelint": "^7.9.0"
+    "stylelint": "^7.12.0 || ~8.0.0"
```
Comment (adidahiya) [inline package.json]: why tilde?

Comment (giladgray) [inline package.json]: first bullet https://github.com/stylelint/stylelint/releases/tag/8.0.0

> A new semantic version policy. The use of the tilde (~) in package.json is now recommended, e.g. "stylelint": "~8.0.0", to guarantee the results of your builds (#1865).

File: package.json [hunk 3]
```diff
@@ -23,15 +23,15 @@
...
   },
   "devDependencies": {
-    "stylelint": "7.9.0",
-    "stylelint-scss": "1.4.3"
+    "stylelint": "8.0.0",
+    "stylelint-scss": "1.5.1"
   },
   "peerDependencies": {
-    "stylelint": "^7.9.0"
+    "stylelint": "^7.12.0 || ~8.0.0"
```
Comment (adidahiya) [inline package.json]: That [semantic versioning policy](https://github.com/stylelint/stylelint/blob/8.0.0/README.md#semantic-versioning-policy) is for applications, not config libraries like this. Basically what you've done here is prevent our apps from upgrading to 8.1, which is not what we want. The new policy isn't entirely novel; like they said it's like other code quality tools, including TSLint. Based on my experience with TSLint I think you should make this `^7.12.0 || `^8.0.0`.

Comment (giladgray) [inline package.json]: ugh those operators are confusing

**Pull request merged**